### PR TITLE
Add freezing to UI

### LINKED ISSUE #286: Add freeze attribute for courses and concepts to prevent editing

### PR DESCRIPTION
**Pull request by tulir**
Fixes #286

### CODE REVIEW

File: backend/src/resolvers/Mutation/Course.js [hunk 1]
```diff
@@ -63,8 +67,8 @@ const CourseQueries = {
     const tagsToDelete = oldTags
       .filter(oldTag => !tags.find(tag => tag.id === oldTag.id))
```
Comment (tulir) [inline backend/src/resolvers/Mutation/Course.js]: Should check if `tags` exist

File: frontend/src/dialogs/Dialog.js [hunk 2]
```diff
@@ -185,7 +185,8 @@ const Dialog = ({ contextRef }) => {
                   isMulti={field.isMultiSelect}
                 />
               } else if (field.type === 'checkbox') {
-                return <FormControl fullWidth key={field.name}>
+                return <FormControl style={{ verticalAlign: 'middle', marginRight: '12px' }}
+                  key={field.name}>
```
Comment (tulir) [inline frontend/src/dialogs/Dialog.js]: This should maybe be wrapped better

```js
<FormControl
  style=...
  key=...
>
  children
</FormControl>
```

File: frontend/src/views/manager/CourseEditor.js [hunk 3]
```diff
@@ -382,6 +403,20 @@ const CreateConcept = ({ submit, defaultValues = {}, action = 'Create', cancel }
...
+                checked={input.frozen}
+                onChange={evt => setInput({ ...input, frozen: evt.target.checked })}
+                value='frozen'
+                color='primary'
+              />
+            }
+            label='Frozen'
+          />
+        </FormControl>
+        </>
```
Comment (tulir) [inline frontend/src/views/manager/CourseEditor.js]: This indentation doesn't look right

File: frontend/src/views/mapper/ActiveCourse.js [hunk 4]
```diff
@@ -120,9 +120,12 @@ const ActiveCourse = ({
           )) : <MenuItem value={course.id}>{course.name}</MenuItem>}
         </Select>
         <div className={classes.titleEditWrapper}>
-          <IconButton onClick={() =>
-            openEditCourseDialog(course.id, course.name, course.official, course.tags)
-          }>
+
```
Comment (tulir) [inline frontend/src/views/mapper/ActiveCourse.js]: Blank line

```suggestion
```

File: frontend/src/views/manager/CourseEditor.js [hunk 5]
```diff
@@ -283,7 +303,8 @@ const initialState = {
   description: '',
   tags: [],
   bloomTag: '',
-  official: undefined
+  official: undefined,
```
Comment (d471061c) [inline frontend/src/views/manager/CourseEditor.js]: Could be set to true as default in project templates.

**Pull request merged**